website/settings/defaults.py

```diff
@@ -1832,3 +1832,19 @@ def parent_dir(path):
 }
 
 PREPRINTS_ASSETS = '/static/img/preprints_assets/'
+
+SOCIAL_FIELDS = {
```

Comment:
Was it necessary to move this? This isn't really a setting, so putting it in settings seems inappropriate.